src/main/java/frc/trigon/robot/subsystems/sideshooter/SideShooter.java

```diff
@@ -0,0 +1,80 @@
+package frc.trigon.robot.subsystems.sideshooter;
+
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.lang.annotation.Target;
+
+public class SideShooter extends SubsystemBase {
```

Comment:
Reformat class (ctrl + alt + L) 
And for next time, install save actions like I mentioned in the group